ocs_ci/deployment/cnv.py

```diff
@@ -792,3 +797,92 @@ def uninstall_cnv(self, check_cnv_installed=True):
 
         logger.info("Removing the openshift virtualization CRDs")
         self.remove_crds()
+
+    def cnv_patch_subscription(self, patch):
```

Comment:
Please make use of the patch method in. class OCP.

```
hyperconverged_subs = OCP(kind=constants.SUBSCRIPTION_WITH_ACM, namespace=self.namespace, resource_name=constants.KUBEVIRT_HYPERCONVERGED)
        hyperconverged_subs.patch(params=patch, format_type="merge")
```

This method itself can be removed if this is handled from where this is called.